plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/params/BuildImageParams.java

```diff
@@ -182,6 +188,9 @@ public BuildImageParams withFiles(@NotNull File... files) {
      *         if {@code files} is null
      */
     public BuildImageParams addFiles(@NotNull File... files) {
+        if (remote != null) {
```

Comment:
withFiles will be called before becase it is used in create method. If create with remote is called then https://github.com/eclipse/che/pull/1705/files/46da34dd88373c9b7292e0d0d1aaecdf20fe4ace#diff-3b42f0904980953ecdfb9059553a7736R194 will fail